tests/conftest.py

```diff
@@ -8,11 +8,11 @@
 from briefcase.config import AppConfig
 from briefcase.console import Printer
 
-# Stop Rich from inserting line breaks in to long lines of text.
-# Rich does this to prevent individual words being split between
-# two lines in the terminal...however, these additional line breaks
-# cause some tests to fail unexpectedly.
-Printer.console.soft_wrap = True
```

Comment:
Setting `soft_wrap=True` is no longer necessary since we always set it to `True` in `console.py`.